wordpress-mobile/gutenberg-mobile
# Correct blockWidth

Fixes the issue with missing cog button in nested Button block

gb pr: https://github.com/WordPress/gutenberg/pull/28635

To test:
1. Open an app
2. Add `Buttons` block
3. Save a post
4. Open it again
5. Focus the `Button` within `Buttons`
6. Expect settings buttons is visible (cog)


PR submission checklist:
